flutter-candidate.txt

```diff
@@ -1 +1 @@
-3.17.0-9.0.pre
```

Comment:
we shouldn't update this file manually - it is automatically updated by a github workflow when a new candidate branch is ready. The workflow runs 2x per day.

Reply:
Got it, let's wait until it automatically updates to the needed version then.